ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-ospf.j2

```diff
@@ -4,8 +4,8 @@
 
 ### Router OSPF Summary
 
-| Process ID | Router ID | Default Passive Interface | No Passive Interface | BFD | Max LSA | Default Information Originate | Log Adjacency Changes Detail | Auto Cost Reference Bandwidth | Maximum Paths | MPLS LDP Sync Default | Distribute List In |
-| ---------- | --------- | ------------------------- | -------------------- | --- | ------- | ----------------------------- | ---------------------------- | ----------------------------- | ------------- | --------------------- | ------------------ |
+| Process ID | Router ID | Default Passive Interface | No Passive Interface | BFD | BFD Any | Max LSA | Default Information Originate | Log Adjacency Changes Detail | Auto Cost Reference Bandwidth | Maximum Paths | MPLS LDP Sync Default | Distribute List In |
```

Comment:
Given that you are changing this template you should be running in the molecule folder
```
make refresh-facts
```

as it will impact more than just the `eos_cli_config_gen` scenarios but most of them (that is why you have so many failure on the pipeline)

That being said - I am wondering if we should not rather separate the OSPF BFD information in its own table if we start to have multiple of them @ClausHolbechArista  thoughts?

Reply:
1. We should not add a new column for - what I guess is - a rarely used field. If we want to show it in the table, we could just add `enabled (any state)` to the `BFD` cell value.

Reply:
done.